src/features/projectsV2/ProjectsSection.tsx

```diff
@@ -1,25 +1,77 @@
-import ProjectSnippet from './components/ProjectSnippet';
-import style from './styles/ProjectsSection.module.scss';
-import { useProjects } from './ProjectsContext';
 import Skeleton from 'react-loading-skeleton';
 import 'react-loading-skeleton/dist/skeleton.css';
+import styles from './ProjectsSection.module.scss';
+import { useProjects } from './ProjectsContext';
+import ProjectListControls from './ProjectListControls';
+import ProjectListControlForMobile from './ProjectListControls/ProjectListControlForMobile';
+import { SetState } from '../common/types/common';
+import ProjectList from './ProjectList';
+import { ViewMode } from '../../../pages/_app';
+
+interface ProjectsSectionProps {
+  selectedMode: ViewMode;
+  setSelectedMode: SetState<ViewMode>;
+  isMobile: boolean;
+}
 
-const ProjectsSection = () => {
-  const { projects, isLoading, isError } = useProjects();
-  if (isLoading || isError) {
-    return <Skeleton className={style.projectSectionSkeleton} />;
+const ProjectsSection = ({
+  selectedMode,
+  setSelectedMode,
+  isMobile,
+}: ProjectsSectionProps) => {
+  const {
+    projects,
+    topProjects,
+    selectedClassification,
+    setSelectedClassification,
+    filteredProjects,
+    tabSelected,
+    setTabSelected,
+    debouncedSearchValue,
+    setDebouncedSearchValue,
+    isSearching,
+    setIsSearching,
```

Comment:
Why did we need `isSearching` state in the context, when `debouncedSearchValue` is also available?